README.md

```diff
@@ -180,7 +147,7 @@ DOCS_DEPLOYMENT=release make clean html
 
 ## Built With
 
-* [Eclipse](http://www.eclipse.org/downloads/packages/eclipse-ide-java-ee-developers/oxygen1a) - IDE
+* [Eclipse](https://www.eclipse.org) - IDE
```

Comment:
do we really need to mention the IDE? I mean, it's possible to use other IDEs just fine, and not even all of us use eclipse regularly :wink:

Reply:
true, in the meantime we have more collaborators that also use different IDEs :) 
I'll remove that part about the IDE.